src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -326,7 +326,7 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
         /// <param name="csvStream">stream of CSV data to be write out</param>
         /// <param name="separator">column separator</param>
         /// <param name="header">has a header or not</param>
-        /// <param name="encoding">the character encoding. Defaults to UTF8 if not specified</param>
+        /// <param name="encoding">the character encoding. Defaults to ASCII if not specified</param>
```

Comment:
We should keep the same docs, and change the code. It is wrong for ReadCsv to default to Utf8 and WriteCsv to default to Ascii. Let's use Utf8 for both defaults.